core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2566,7 +2579,7 @@ public String toString() {
           .setScope(Scope.MASTER)
           .build();
   public static final PropertyKey MASTER_JOURNAL_CHECKPOINT_PERIOD_ENTRIES =
-      intBuilder(Name.MASTER_JOURNAL_CHECKPOINT_PERIOD_ENTRIES)
+      longBuilder(Name.MASTER_JOURNAL_CHECKPOINT_PERIOD_ENTRIES)
```

Comment:
I don't think these property keys need to be long. Integer can already go up to 2billion+, which is probably larger than what our system can handle already. If we really need a LONG type property key, let's add that. But these keys should not be of type LONG.